### PR TITLE
[RISC-V] Add riscv64 arch to tracing/eventpipe/processinfo tests

### PR DESCRIPTION
**Pull request by gbalykov**
With TieredCompilation tracing/eventpipe/processinfo tests are skipped by theirs .sh launchers. Disabling TieredCompilation leads to fails like:

```
               Unhandled exception. System.Exception: OS must match current Operating System. Expected: "Unknown", Received: "riscv64"^M
                  at Tracing.Tests.Common.Utils.Assert(Boolean predicate, String message)^M
                  at Tracing.Tests.ProcessInfoValidation.ProcessInfoValidation.Main()^M
```

This patch adds RiscV64 and makes pass tracing/eventpipe/processinfo tests with `DOTNET_TieredCompilation="0"` option. Tests influenced:

```
tracing/eventpipe/processinfo/processinfo/processinfo.sh
tracing/eventpipe/processinfo2/processinfo2/processinfo2.sh
tracing/eventpipe/processinfo3/processinfo3/processinfo3.sh
```

cc @dotnet/samsung. Part of https://github.com/dotnet/runtime/issues/84834.

cc @shushanhf @LuckyXu-HF seems Loongarch64 needs same changes.

### CODE REVIEW

**Comment by risc-vv** [comment]
### RISC-V test results for starfive-prio1-checked (`Checked` build): 9407 / 9408 (99.99%)
<details>
<summary>details</summary>

GIT: ef55f8ef5c0d2bf8fcc206f4d69543a6da47d584
Build mode: `Checked`
Links:
1. [tests_results.xml](https://gist.githubusercontent.com/risc-vv/d9d9ad25b895735e1763a3c3e247dbe1/raw/83f754b3e0a61cc664550576a78d9bf28beb9ff1/c7d66ca13c585a3a2bd7df343245a4b37be1ba2fc5412a5ad370497aaf799c90.xml)
2. [tests_output.tar.gz](http://127.0.0.1/pipelines/starfive-prio1-checked/14/run-tests/1/ALL/testoutput/test_datails.Checked.tar.gz)
3. [coreclr/prio1 tests binary](http://127.0.0.1/pipelines/build-runtime-checked/35/build-tests-prio1/1/build/coreclr.Checked.prio1.tar.gz)
4. [clr+libs+host binary](http://127.0.0.1/pipelines/build-runtime-checked/35/build-core/1/build/artifacts.Checked.tar.gz)
5. [testhost binary](http://127.0.0.1/pipelines/build-runtime-checked/35/build-core/1/build/testhost.Checked.tar.gz)
```

=======================
      passed: 9407
      failed: 1
     skipped: 42
      killed: 0
------------------------
  TOTAL libs: 9450
 TOTAL tests: 9450
   REAL time: 2h 50min 51s 354ms
=======================

```
<details>
<summary>failed tests</summary>

```
[712.870s] readytorun.coreroot_determinism.coreroot_determinism.coreroot_determinism
    coreroot_determinism.sh
    [exitcode-  1]: unknown error

```
</details>
<details>
<summary>killed tests</summary>

```

```
</details>
<details>
<summary>skipped tests</summary>

```
[0.000s] readytorun.DynamicMethodGCStress.DynamicMethodGCStress.DynamicMethodGCStress
    DynamicMethodGCStress.sh
[0.000s] readytorun.GenericCycleDetection.Depth3Test.Depth3Test
    Depth3Test.sh
[0.000s] GC.Features.LOHFragmentation.lohfragmentation.lohfragmentation
    lohfragmentation.sh
[0.000s] GC.Features.BackgroundGC.foregroundgc.foregroundgc
    foregroundgc.sh
[0.000s] GC.Coverage.271010.271010
    271010.sh
[0.000s] GC.LargeMemory.Allocation.finalizertest.finalizertest
    finalizertest.sh
[0.000s] GC.LargeMemory.API.gc.collect.collect
    collect.sh
[0.000s] GC.LargeMemory.API.gc.reregisterforfinalize.reregisterforfinalize
    reregisterforfinalize.sh
[0.000s] GC.LargeMemory.API.gc.getgeneration.getgeneration
    getgeneration.sh
[0.000s] GC.Regressions.dev10bugs.536168.536168.536168
    536168.sh
[0.000s] GC.Regressions.Github.Runtime_76219.Runtime_76219.Runtime_76219
    Runtime_76219.sh
[0.000s] JIT.Regression.CLR-x86-JIT.V1-M11-Beta1.b41391.b41391.b41391
    b41391.sh
[0.000s] JIT.Regression.CLR-x86-JIT.V1-M12-Beta2.b37646.b37646.b37646
    b37646.sh
[0.000s] JIT.Regression.CLR-x86-JIT.V1-M12-Beta2.b31746.b31746.b31746
    b31746.sh
[0.000s] JIT.Regression.CLR-x86-JIT.V1-M12-Beta2.b41852.b41852.b41852
    b41852.sh
[0.000s] JIT.Regression.CLR-x86-JIT.V1-M09.5-PDC.b16423.b16423.b16423
    b16423.sh
[0.000s] JIT.Regression.JitBlue.Runtime_57606.Runtime_57606.Runtime_57606
    Runtime_57606.sh
[0.000s] JIT.Regression.CLR-x86-EJIT.V1-M12-Beta2.b26323.b26323.b26323
    b26323.sh
[0.000s] JIT.Directed.arglist.vararg_TargetUnix.vararg_TargetUnix
    vararg_TargetUnix.sh
[0.000s] JIT.Directed.rvastatics.RVAOrderingTest.RVAOrderingTest
    RVAOrderingTest.sh
[0.000s] JIT.Directed.PREFIX.volatile.1.arglist_Target_64BIT_volatile.arglist_Target_64BIT_volatile
    arglist_Target_64BIT_volatile.sh
[0.000s] JIT.Directed.PREFIX.unaligned.4.arglist_Target_64BIT_unaligned_4.arglist_Target_64BIT_unaligned_4
    arglist_Target_64BIT_unaligned_4.sh
[0.000s] JIT.Directed.PREFIX.unaligned.2.arglist_Target_64BIT_unaligned_2.arglist_Target_64BIT_unaligned_2
    arglist_Target_64BIT_unaligned_2.sh
[0.000s] JIT.Directed.PREFIX.unaligned.1.arglist_Target_64BIT_unaligned_1.arglist_Target_64BIT_unaligned_1
    arglist_Target_64BIT_unaligned_1.sh
[0.000s] JIT.jit64.mcc.interop.mcc_i00.mcc_i00
    mcc_i00.sh
[0.000s] JIT.opt.SSA.MemorySsa.MemorySsa
    MemorySsa.sh
[0.000s] JIT.opt.ValueNumbering.ExposedLocalsNumbering.ExposedLocalsNumbering
    ExposedLocalsNumbering.sh
[0.000s] JIT.Methodical.Coverage.arglist_pos.arglist_pos
    arglist_pos.sh
[0.000s] JIT.Methodical.refany.seq_d.seq_d
    seq_d.sh
[0.000s] JIT.Methodical.refany.seq_r.seq_r
    seq_r.sh
[0.000s] Loader.binding.tracing.BinderTracingTest.Basic.BinderTracingTest.Basic
    BinderTracingTest.Basic.sh
[0.000s] Interop.MonoAPI.MonoMono.Thunks.Thunks
    Thunks.sh
[0.000s] Interop.MonoAPI.MonoMono.PInvokeDetach.PInvokeDetach
    PInvokeDetach.sh
[0.000s] Interop.MonoAPI.MonoMono.InstallEHCallback.InstallEHCallback
    InstallEHCallback.sh
[0.000s] tracing.eventpipe.eventsourceerror.eventsourceerror.eventsourceerror
    eventsourceerror.sh
[0.000s] baseservices.finalization.CriticalFinalizer.CriticalFinalizer
    CriticalFinalizer.sh
[0.000s] baseservices.mono.runningmono.runningmono
    runningmono.sh
[0.000s] baseservices.threading.regressions.2164.foreground-shutdown.foreground-shutdown
    foreground-shutdown.sh
[0.000s] baseservices.exceptions.stackoverflow.stackoverflowtester.stackoverflowtester
    stackoverflowtester.sh
[0.000s] baseservices.exceptions.StackTracePreserve.StackTracePreserveTests.StackTracePreserveTests
    StackTracePreserveTests.sh
[0.000s] baseservices.typeequivalence.istypeequivalent.istypeequivalent.istypeequivalent
    istypeequivalent.sh
[0.000s] baseservices.typeequivalence.signatures.nopiatestil.nopiatestil
    nopiatestil.sh

```
</details>
</details>


[Full report on gist](https://gist.githubusercontent.com/risc-vv/65acf2f51dd5aa204db4c0f694ff2450/raw/8c90c382e17d0df4adfdd15212a5654ce2991794/c7d66ca13c585a3a2bd7df343245a4b37be1ba2fc5412a5ad370497aaf799c90.md)

**Comment by risc-vv** [comment]
### RISC-V test results for qemu-prio1-checked (`Checked` build): 9404 / 9408 (99.96%)
<details>
<summary>details</summary>

GIT: ef55f8ef5c0d2bf8fcc206f4d69543a6da47d584
Build mode: `Checked`
Links:
1. [tests_results.xml](https://gist.githubusercontent.com/risc-vv/ed599361c99fa993177373a938b3c5c3/raw/825753024052231c7a86e92ba16567c63e94e68b/c7d66ca13c585a3a2bd7df343245a4b37be1ba2fc5412a5ad370497aaf799c90.xml)
2. [tests_output.tar.gz](http://127.0.0.1/pipelines/qemu-prio1-checked/14/run-tests/1/ALL/testoutput/test_datails.Checked.tar.gz)
3. [coreclr/prio1 tests binary](http://127.0.0.1/pipelines/build-runtime-checked/35/build-tests-prio1/1/build/coreclr.Checked.prio1.tar.gz)
4. [clr+libs+host binary](http://127.0.0.1/pipelines/build-runtime-checked/35/build-core/1/build/artifacts.Checked.tar.gz)
5. [testhost binary](http://127.0.0.1/pipelines/build-runtime-checked/35/build-core/1/build/testhost.Checked.tar.gz)
```

=======================
      passed: 9404
      failed: 3
     skipped: 42
      killed: 1
------------------------
  TOTAL libs: 9450
 TOTAL tests: 9450
   REAL time: 1h 19min 31s 854ms
=======================

```
<details>
<summary>failed tests</summary>

```
[57.740s] profiler.unittest.releaseondetach.releaseondetach
    releaseondetach.sh
    [exitcode-134]: SIGABRT Abort. Managed or native assert, or runtime check such as heap corruption, caused call to abort(). Core dumped.
[38.140s] JIT.Directed.debugging.debuginfo.tester.tester
    tester.sh
    [exitcode-  -1]: unknown error
[735.570s] readytorun.coreroot_determinism.coreroot_determinism.coreroot_determinism
    coreroot_determinism.sh
    [exitcode-  1]: unknown error

```
</details>
<details>
<summary>killed tests</summary>

```
[1802.079s] profiler.eventpipe.reverse_startup.reverse_startup
    reverse_startup.sh

```
</details>
<details>
<summary>skipped tests</summary>

```
[0.000s] Interop.MonoAPI.MonoMono.InstallEHCallback.InstallEHCallback
    InstallEHCallback.sh
[0.000s] Interop.MonoAPI.MonoMono.PInvokeDetach.PInvokeDetach
    PInvokeDetach.sh
[0.000s] Interop.MonoAPI.MonoMono.Thunks.Thunks
    Thunks.sh
[0.000s] tracing.eventpipe.eventsourceerror.eventsourceerror.eventsourceerror
    eventsourceerror.sh
[0.000s] JIT.Directed.arglist.vararg_TargetUnix.vararg_TargetUnix
    vararg_TargetUnix.sh
[0.000s] JIT.Directed.rvastatics.RVAOrderingTest.RVAOrderingTest
    RVAOrderingTest.sh
[0.000s] JIT.Directed.PREFIX.volatile.1.arglist_Target_64BIT_volatile.arglist_Target_64BIT_volatile
    arglist_Target_64BIT_volatile.sh
[0.000s] JIT.Directed.PREFIX.unaligned.4.arglist_Target_64BIT_unaligned_4.arglist_Target_64BIT_unaligned_4
    arglist_Target_64BIT_unaligned_4.sh
[0.000s] JIT.Directed.PREFIX.unaligned.1.arglist_Target_64BIT_unaligned_1.arglist_Target_64BIT_unaligned_1
    arglist_Target_64BIT_unaligned_1.sh
[0.000s] JIT.Directed.PREFIX.unaligned.2.arglist_Target_64BIT_unaligned_2.arglist_Target_64BIT_unaligned_2
    arglist_Target_64BIT_unaligned_2.sh
[0.000s] JIT.opt.ValueNumbering.ExposedLocalsNumbering.ExposedLocalsNumbering
    ExposedLocalsNumbering.sh
[0.000s] JIT.opt.SSA.MemorySsa.MemorySsa
    MemorySsa.sh
[0.000s] JIT.Methodical.refany.seq_d.seq_d
    seq_d.sh
[0.000s] JIT.Methodical.refany.seq_r.seq_r
    seq_r.sh
[0.000s] JIT.Methodical.Coverage.arglist_pos.arglist_pos
    arglist_pos.sh
[0.000s] JIT.Regression.CLR-x86-JIT.V1-M12-Beta2.b37646.b37646.b37646
    b37646.sh
[0.000s] JIT.Regression.CLR-x86-JIT.V1-M12-Beta2.b41852.b41852.b41852
    b41852.sh
[0.000s] JIT.Regression.CLR-x86-JIT.V1-M12-Beta2.b31746.b31746.b31746
    b31746.sh
[0.000s] JIT.Regression.CLR-x86-JIT.V1-M11-Beta1.b41391.b41391.b41391
    b41391.sh
[0.000s] JIT.Regression.CLR-x86-JIT.V1-M09.5-PDC.b16423.b16423.b16423
    b16423.sh
[0.000s] JIT.Regression.JitBlue.Runtime_57606.Runtime_57606.Runtime_57606
    Runtime_57606.sh
[0.000s] JIT.Regression.CLR-x86-EJIT.V1-M12-Beta2.b26323.b26323.b26323
    b26323.sh
[0.000s] JIT.jit64.mcc.interop.mcc_i00.mcc_i00
    mcc_i00.sh
[0.000s] Loader.binding.tracing.BinderTracingTest.Basic.BinderTracingTest.Basic
    BinderTracingTest.Basic.sh
[0.000s] readytorun.GenericCycleDetection.Depth3Test.Depth3Test
    Depth3Test.sh
[0.000s] readytorun.DynamicMethodGCStress.DynamicMethodGCStress.DynamicMethodGCStress
    DynamicMethodGCStress.sh
[0.000s] GC.LargeMemory.API.gc.reregisterforfinalize.reregisterforfinalize
    reregisterforfinalize.sh
[0.000s] GC.LargeMemory.API.gc.getgeneration.getgeneration
    getgeneration.sh
[0.000s] GC.LargeMemory.API.gc.collect.collect
    collect.sh
[0.000s] GC.LargeMemory.Allocation.finalizertest.finalizertest
    finalizertest.sh
[0.000s] GC.Regressions.dev10bugs.536168.536168.536168
    536168.sh
[0.000s] GC.Regressions.Github.Runtime_76219.Runtime_76219.Runtime_76219
    Runtime_76219.sh
[0.000s] GC.Features.BackgroundGC.foregroundgc.foregroundgc
    foregroundgc.sh
[0.000s] GC.Features.LOHFragmentation.lohfragmentation.lohfragmentation
    lohfragmentation.sh
[0.000s] GC.Coverage.271010.271010
    271010.sh
[0.000s] baseservices.threading.regressions.2164.foreground-shutdown.foreground-shutdown
    foreground-shutdown.sh
[0.000s] baseservices.mono.runningmono.runningmono
    runningmono.sh
[0.000s] baseservices.typeequivalence.signatures.nopiatestil.nopiatestil
    nopiatestil.sh
[0.000s] baseservices.typeequivalence.istypeequivalent.istypeequivalent.istypeequivalent
    istypeequivalent.sh
[0.000s] baseservices.finalization.CriticalFinalizer.CriticalFinalizer
    CriticalFinalizer.sh
[0.000s] baseservices.exceptions.stackoverflow.stackoverflowtester.stackoverflowtester
    stackoverflowtester.sh
[0.000s] baseservices.exceptions.StackTracePreserve.StackTracePreserveTests.StackTracePreserveTests
    StackTracePreserveTests.sh

```
</details>
</details>


[Full report on gist](https://gist.githubusercontent.com/risc-vv/b960e2e3cec84395e23a0623dfab08ec/raw/3ac02c0dace1e27d55017c2a35081bb7f7cf2039/c7d66ca13c585a3a2bd7df343245a4b37be1ba2fc5412a5ad370497aaf799c90.md)

File: src/tests/tracing/eventpipe/processinfo3/processinfo3.cs [hunk 1]
```diff
@@ -233,10 +233,11 @@ public static void TestEntryPoint()
                 Architecture.X64 => "x64",
                 Architecture.Arm => "arm32",
                 Architecture.Arm64 => "arm64",
+                Architecture.RiscV64 => "riscv64",
```
Comment (am11) [inline src/tests/tracing/eventpipe/processinfo3/processinfo3.cs:236]: Please also add loongarch64 while updating these.

Comment (gbalykov) [reply src/tests/tracing/eventpipe/processinfo3/processinfo3.cs:236]: I've added loongarch64 too, but please note that I've not verified that test passes on it now. @shushanhf @LuckyXu-HF please take a look.

Comment (am11) [reply src/tests/tracing/eventpipe/processinfo3/processinfo3.cs:236]: Thanks. Yes, it's funny that either that test is suppressed or it throws anyway. We have these unnecessary platform maps in too many places which add no value (just another string to update for the next platform).

**Comment by risc-vv** [comment]
### RISC-V test results for starfive-prio1-checked (`Checked` build): 9407 / 9408 (99.99%)
<details>
<summary>details</summary>

GIT: 2e4bb32503844f418bb84e61a7932be4b94d911e
Build mode: `Checked`
Links:
1. [tests_results.xml](https://gist.githubusercontent.com/risc-vv/8939dc4334d08288afb4f8d63c416e67/raw/a579f4154c47d885df328674165503d3c0662e77/8e828d92d377f7039481827c4c2814f2dcc4f80368a629d3ceb97a20870d7f04.xml)
2. [tests_output.tar.gz](http://127.0.0.1/pipelines/starfive-prio1-checked/15/run-tests/1/ALL/testoutput/test_datails.Checked.tar.gz)
3. [coreclr/prio1 tests binary](http://127.0.0.1/pipelines/build-runtime-checked/36/build-tests-prio1/1/build/coreclr.Checked.prio1.tar.gz)
4. [clr+libs+host binary](http://127.0.0.1/pipelines/build-runtime-checked/36/build-core/1/build/artifacts.Checked.tar.gz)
5. [testhost binary](http://127.0.0.1/pipelines/build-runtime-checked/36/build-core/1/build/testhost.Checked.tar.gz)
```

=======================
      passed: 9407
      failed: 1
     skipped: 42
      killed: 0
------------------------
  TOTAL libs: 9450
 TOTAL tests: 9450
   REAL time: 2h 51min 16s 384ms
=======================

```
<details>
<summary>failed tests</summary>

```
[714.250s] readytorun.coreroot_determinism.coreroot_determinism.coreroot_determinism
    coreroot_determinism.sh
    [exitcode-  1]: unknown error

```
</details>
<details>
<summary>killed tests</summary>

```

```
</details>
<details>
<summary>skipped tests</summary>

```
[0.000s] readytorun.DynamicMethodGCStress.DynamicMethodGCStress.DynamicMethodGCStress
    DynamicMethodGCStress.sh
[0.000s] readytorun.GenericCycleDetection.Depth3Test.Depth3Test
    Depth3Test.sh
[0.000s] GC.Features.LOHFragmentation.lohfragmentation.lohfragmentation
    lohfragmentation.sh
[0.000s] GC.Features.BackgroundGC.foregroundgc.foregroundgc
    foregroundgc.sh
[0.000s] GC.Coverage.271010.271010
    271010.sh
[0.000s] GC.LargeMemory.Allocation.finalizertest.finalizertest
    finalizertest.sh
[0.000s] GC.LargeMemory.API.gc.collect.collect
    collect.sh
[0.000s] GC.LargeMemory.API.gc.reregisterforfinalize.reregisterforfinalize
    reregisterforfinalize.sh
[0.000s] GC.LargeMemory.API.gc.getgeneration.getgeneration
    getgeneration.sh
[0.000s] GC.Regressions.dev10bugs.536168.536168.536168
    536168.sh
[0.000s] GC.Regressions.Github.Runtime_76219.Runtime_76219.Runtime_76219
    Runtime_76219.sh
[0.000s] JIT.Regression.CLR-x86-JIT.V1-M11-Beta1.b41391.b41391.b41391
    b41391.sh
[0.000s] JIT.Regression.CLR-x86-JIT.V1-M12-Beta2.b37646.b37646.b37646
    b37646.sh
[0.000s] JIT.Regression.CLR-x86-JIT.V1-M12-Beta2.b31746.b31746.b31746
    b31746.sh
[0.000s] JIT.Regression.CLR-x86-JIT.V1-M12-Beta2.b41852.b41852.b41852
    b41852.sh
[0.000s] JIT.Regression.CLR-x86-JIT.V1-M09.5-PDC.b16423.b16423.b16423
    b16423.sh
[0.000s] JIT.Regression.JitBlue.Runtime_57606.Runtime_57606.Runtime_57606
    Runtime_57606.sh
[0.000s] JIT.Regression.CLR-x86-EJIT.V1-M12-Beta2.b26323.b26323.b26323
    b26323.sh
[0.000s] JIT.Directed.arglist.vararg_TargetUnix.vararg_TargetUnix
    vararg_TargetUnix.sh
[0.000s] JIT.Directed.rvastatics.RVAOrderingTest.RVAOrderingTest
    RVAOrderingTest.sh
[0.000s] JIT.Directed.PREFIX.volatile.1.arglist_Target_64BIT_volatile.arglist_Target_64BIT_volatile
    arglist_Target_64BIT_volatile.sh
[0.000s] JIT.Directed.PREFIX.unaligned.4.arglist_Target_64BIT_unaligned_4.arglist_Target_64BIT_unaligned_4
    arglist_Target_64BIT_unaligned_4.sh
[0.000s] JIT.Directed.PREFIX.unaligned.2.arglist_Target_64BIT_unaligned_2.arglist_Target_64BIT_unaligned_2
    arglist_Target_64BIT_unaligned_2.sh
[0.000s] JIT.Directed.PREFIX.unaligned.1.arglist_Target_64BIT_unaligned_1.arglist_Target_64BIT_unaligned_1
    arglist_Target_64BIT_unaligned_1.sh
[0.000s] JIT.jit64.mcc.interop.mcc_i00.mcc_i00
    mcc_i00.sh
[0.000s] JIT.opt.SSA.MemorySsa.MemorySsa
    MemorySsa.sh
[0.000s] JIT.opt.ValueNumbering.ExposedLocalsNumbering.ExposedLocalsNumbering
    ExposedLocalsNumbering.sh
[0.000s] JIT.Methodical.Coverage.arglist_pos.arglist_pos
    arglist_pos.sh
[0.000s] JIT.Methodical.refany.seq_d.seq_d
    seq_d.sh
[0.000s] JIT.Methodical.refany.seq_r.seq_r
    seq_r.sh
[0.000s] Loader.binding.tracing.BinderTracingTest.Basic.BinderTracingTest.Basic
    BinderTracingTest.Basic.sh
[0.000s] Interop.MonoAPI.MonoMono.Thunks.Thunks
    Thunks.sh
[0.000s] Interop.MonoAPI.MonoMono.PInvokeDetach.PInvokeDetach
    PInvokeDetach.sh
[0.000s] Interop.MonoAPI.MonoMono.InstallEHCallback.InstallEHCallback
    InstallEHCallback.sh
[0.000s] tracing.eventpipe.eventsourceerror.eventsourceerror.eventsourceerror
    eventsourceerror.sh
[0.000s] baseservices.finalization.CriticalFinalizer.CriticalFinalizer
    CriticalFinalizer.sh
[0.000s] baseservices.mono.runningmono.runningmono
    runningmono.sh
[0.000s] baseservices.threading.regressions.2164.foreground-shutdown.foreground-shutdown
    foreground-shutdown.sh
[0.000s] baseservices.exceptions.stackoverflow.stackoverflowtester.stackoverflowtester
    stackoverflowtester.sh
[0.000s] baseservices.exceptions.StackTracePreserve.StackTracePreserveTests.StackTracePreserveTests
    StackTracePreserveTests.sh
[0.000s] baseservices.typeequivalence.istypeequivalent.istypeequivalent.istypeequivalent
    istypeequivalent.sh
[0.000s] baseservices.typeequivalence.signatures.nopiatestil.nopiatestil
    nopiatestil.sh

```
</details>
</details>


[Full report on gist](https://gist.githubusercontent.com/risc-vv/5495f7ba2ee43478de63bae92ed24ced/raw/d0a81f28555392a5a964c6459b7e454a9ca0419c/8e828d92d377f7039481827c4c2814f2dcc4f80368a629d3ceb97a20870d7f04.md)

**Review by tomeksowi — APPROVED** [review]

**Review by clamp03 — APPROVED** [review]

File: src/tests/tracing/eventpipe/processinfo3/processinfo3.cs (hunk 1 above)
Comment (LuckyXu-HF) [reply src/tests/tracing/eventpipe/processinfo3/processinfo3.cs:236]: Thank you for heads up. I have verified on LA64 and these are also suitable for LA64.
LGTM, thanks very much.

**Review by LuckyXu-HF — APPROVED** [review]

File: src/tests/tracing/eventpipe/processinfo/processinfo.cs [hunk 2]
```diff
@@ -224,14 +224,16 @@ public static void TestEntryPoint()
                 Architecture.X64 => "x64",
                 Architecture.Arm => "arm32",
                 Architecture.Arm64 => "arm64",
+                Architecture.RiscV64 => "riscv64",
```
Comment (jkotas) [inline src/tests/tracing/eventpipe/processinfo/processinfo.cs:227]:
```
string expectedArchValue = RuntimeInformation.ProcessArchitecture switch
{
    Architecture.Arm => "arm32",
    // All other architectures match the enum member name
    _ => RuntimeInformation.ProcessArchitecture.ToString().ToLowerInvariant()
}
```
Can we make this more succinct and reduce need for updates with every new architecture?

Comment (gbalykov) [reply src/tests/tracing/eventpipe/processinfo/processinfo.cs]: Thanks, I've updated PR

**Comment by risc-vv** [comment]
### RISC-V test results for qemu-prio1-checked (`Checked` build): 9404 / 9408 (99.96%)
<details>
<summary>details</summary>

GIT: 2e4bb32503844f418bb84e61a7932be4b94d911e
Build mode: `Checked`
Links:
1. [tests_results.xml](https://gist.githubusercontent.com/risc-vv/8a63d77874b106009c0fd125bd4abd28/raw/503d8e59645f769e43a052698b2a941034c4ac29/8e828d92d377f7039481827c4c2814f2dcc4f80368a629d3ceb97a20870d7f04.xml)
2. [tests_output.tar.gz](http://127.0.0.1/pipelines/qemu-prio1-checked/15/run-tests/1/ALL/testoutput/test_datails.Checked.tar.gz)
3. [coreclr/prio1 tests binary](http://127.0.0.1/pipelines/build-runtime-checked/36/build-tests-prio1/1/build/coreclr.Checked.prio1.tar.gz)
4. [clr+libs+host binary](http://127.0.0.1/pipelines/build-runtime-checked/36/build-core/1/build/artifacts.Checked.tar.gz)
5. [testhost binary](http://127.0.0.1/pipelines/build-runtime-checked/36/build-core/1/build/testhost.Checked.tar.gz)
```

=======================
      passed: 9404
      failed: 3
     skipped: 42
      killed: 1
------------------------
  TOTAL libs: 9450
 TOTAL tests: 9450
   REAL time: 1h 18min 43s 809ms
=======================

```
<details>
<summary>failed tests</summary>

```
[51.120s] profiler.unittest.releaseondetach.releaseondetach
    releaseondetach.sh
    [exitcode-134]: SIGABRT Abort. Managed or native assert, or runtime check such as heap corruption, caused call to abort(). Core dumped.
[38.430s] JIT.Directed.debugging.debuginfo.tester.tester
    tester.sh
    [exitcode-  -1]: unknown error
[753.230s] readytorun.coreroot_determinism.coreroot_determinism.coreroot_determinism
    coreroot_determinism.sh
    [exitcode-  1]: unknown error

```
</details>
<details>
<summary>killed tests</summary>

```
[1800.374s] profiler.eventpipe.reverse_startup.reverse_startup
    reverse_startup.sh

```
</details>
<details>
<summary>skipped tests</summary>

```
[0.000s] Interop.MonoAPI.MonoMono.InstallEHCallback.InstallEHCallback
    InstallEHCallback.sh
[0.000s] Interop.MonoAPI.MonoMono.PInvokeDetach.PInvokeDetach
    PInvokeDetach.sh
[0.000s] Interop.MonoAPI.MonoMono.Thunks.Thunks
    Thunks.sh
[0.000s] tracing.eventpipe.eventsourceerror.eventsourceerror.eventsourceerror
    eventsourceerror.sh
[0.000s] JIT.Directed.arglist.vararg_TargetUnix.vararg_TargetUnix
    vararg_TargetUnix.sh
[0.000s] JIT.Directed.rvastatics.RVAOrderingTest.RVAOrderingTest
    RVAOrderingTest.sh
[0.000s] JIT.Directed.PREFIX.volatile.1.arglist_Target_64BIT_volatile.arglist_Target_64BIT_volatile
    arglist_Target_64BIT_volatile.sh
[0.000s] JIT.Directed.PREFIX.unaligned.4.arglist_Target_64BIT_unaligned_4.arglist_Target_64BIT_unaligned_4
    arglist_Target_64BIT_unaligned_4.sh
[0.000s] JIT.Directed.PREFIX.unaligned.1.arglist_Target_64BIT_unaligned_1.arglist_Target_64BIT_unaligned_1
    arglist_Target_64BIT_unaligned_1.sh
[0.000s] JIT.Directed.PREFIX.unaligned.2.arglist_Target_64BIT_unaligned_2.arglist_Target_64BIT_unaligned_2
    arglist_Target_64BIT_unaligned_2.sh
[0.000s] JIT.opt.ValueNumbering.ExposedLocalsNumbering.ExposedLocalsNumbering
    ExposedLocalsNumbering.sh
[0.000s] JIT.opt.SSA.MemorySsa.MemorySsa
    MemorySsa.sh
[0.000s] JIT.Methodical.refany.seq_d.seq_d
    seq_d.sh
[0.000s] JIT.Methodical.refany.seq_r.seq_r
    seq_r.sh
[0.000s] JIT.Methodical.Coverage.arglist_pos.arglist_pos
    arglist_pos.sh
[0.000s] JIT.Regression.CLR-x86-JIT.V1-M12-Beta2.b37646.b37646.b37646
    b37646.sh
[0.000s] JIT.Regression.CLR-x86-JIT.V1-M12-Beta2.b41852.b41852.b41852
    b41852.sh
[0.000s] JIT.Regression.CLR-x86-JIT.V1-M12-Beta2.b31746.b31746.b31746
    b31746.sh
[0.000s] JIT.Regression.CLR-x86-JIT.V1-M11-Beta1.b41391.b41391.b41391
    b41391.sh
[0.000s] JIT.Regression.CLR-x86-JIT.V1-M09.5-PDC.b16423.b16423.b16423
    b16423.sh
[0.000s] JIT.Regression.JitBlue.Runtime_57606.Runtime_57606.Runtime_57606
    Runtime_57606.sh
[0.000s] JIT.Regression.CLR-x86-EJIT.V1-M12-Beta2.b26323.b26323.b26323
    b26323.sh
[0.000s] JIT.jit64.mcc.interop.mcc_i00.mcc_i00
    mcc_i00.sh
[0.000s] Loader.binding.tracing.BinderTracingTest.Basic.BinderTracingTest.Basic
    BinderTracingTest.Basic.sh
[0.000s] readytorun.GenericCycleDetection.Depth3Test.Depth3Test
    Depth3Test.sh
[0.000s] readytorun.DynamicMethodGCStress.DynamicMethodGCStress.DynamicMethodGCStress
    DynamicMethodGCStress.sh
[0.000s] GC.LargeMemory.API.gc.reregisterforfinalize.reregisterforfinalize
    reregisterforfinalize.sh
[0.000s] GC.LargeMemory.API.gc.getgeneration.getgeneration
    getgeneration.sh
[0.000s] GC.LargeMemory.API.gc.collect.collect
    collect.sh
[0.000s] GC.LargeMemory.Allocation.finalizertest.finalizertest
    finalizertest.sh
[0.000s] GC.Regressions.dev10bugs.536168.536168.536168
    536168.sh
[0.000s] GC.Regressions.Github.Runtime_76219.Runtime_76219.Runtime_76219
    Runtime_76219.sh
[0.000s] GC.Features.BackgroundGC.foregroundgc.foregroundgc
    foregroundgc.sh
[0.000s] GC.Features.LOHFragmentation.lohfragmentation.lohfragmentation
    lohfragmentation.sh
[0.000s] GC.Coverage.271010.271010
    271010.sh
[0.000s] baseservices.threading.regressions.2164.foreground-shutdown.foreground-shutdown
    foreground-shutdown.sh
[0.000s] baseservices.mono.runningmono.runningmono
    runningmono.sh
[0.000s] baseservices.typeequivalence.signatures.nopiatestil.nopiatestil
    nopiatestil.sh
[0.000s] baseservices.typeequivalence.istypeequivalent.istypeequivalent.istypeequivalent
    istypeequivalent.sh
[0.000s] baseservices.finalization.CriticalFinalizer.CriticalFinalizer
    CriticalFinalizer.sh
[0.000s] baseservices.exceptions.stackoverflow.stackoverflowtester.stackoverflowtester
    stackoverflowtester.sh
[0.000s] baseservices.exceptions.StackTracePreserve.StackTracePreserveTests.StackTracePreserveTests
    StackTracePreserveTests.sh

```
</details>
</details>


[Full report on gist](https://gist.githubusercontent.com/risc-vv/d53ea15246e81a06005bfd2546d843a3/raw/7f99a0013a3912cf97a8cab57f3280a7ada677be/8e828d92d377f7039481827c4c2814f2dcc4f80368a629d3ceb97a20870d7f04.md)

**Review by jkotas — APPROVED** [review]
Thanks

**Pull request merged**